shared/app/rpc-stats.js

```diff
@@ -15,9 +15,11 @@ type State = {
   // mark* means to make it bold for a single render cause it changed
   markIn: boolean,
   markOut: boolean,
+  markEOF: boolean,
   // counts
-  smallInCount: number,
```

Comment:
any reason for the 'small' prefix? Am guessing it's from an older version with a big count?

Reply:
yeah it used to be more complicated